modules/ephemeral-storage-creating-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * work_with_shared_resources/using-shared-resource-csi-driver.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-creating-configmaps-across-namespaces_{context}"]
+= Creating a SharedConfigMap instance
+
+You can create a `SharedConfigMap` CR instance for the `ConfigMap` object that you want to share across namespaces. 
```

Comment:
"You can create a `SharedConfigMap` custom resource (CR) to share a `ConfigMap` object across namespaces."